### PR TITLE
OCPBUGS-24267: Cluster configuration fields are not visible

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Fixes**: 
https://issues.redhat.com/browse/OCPBUGS-24267

**Analysis / Root cause**: 
`TabContent` component was not expected before

**Solution Description**: 
Added `TabContent` component to display the tab contents 

**Screen shots / Gifs for design review**: 
----BEFORE---

https://drive.google.com/file/d/17TrZNE2dY-AH-vUwcsjvC4E8wxiyPb9n/view?usp=drive_link

----AFTER----

https://github.com/openshift/console/assets/102503482/98d27395-57e6-4576-9744-732a2082d0b6

**Unit test coverage report**: 
NA

**Test setup:**
 1.Open '/cluster-configuration' link
 2.Check all the tabs
 
**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/console-app/src/components/cluster-configuration/useClusterConfigurationGroupItems.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import {
```
Comment (vikram-raj) [inline frontend/packages/console-app/src/components/cluster-configuration/useClusterConfigurationGroupItems.ts:1]: Can you rename this filename `useClusterConfigurationGroupItems.ts` as it does not have any hook?

Comment (lokanandaprabhu) [reply frontend/packages/console-app/src/components/cluster-configuration/useClusterConfigurationGroupItems.ts]: Changed it to `getClusterConfigurationGroups`

**Review by vikram-raj — APPROVED** [review]
/lgtm
/approve

**Pull request merged**